lib/CMakeLists.txt

```diff
@@ -660,17 +648,7 @@ write_basic_package_version_file(
   VERSION ${QUDA_VERSION}
   COMPATIBILITY AnyNewerVersion)
 
-export(
-  EXPORT qudaTargets
-  FILE "${CMAKE_CURRENT_BINARY_DIR}/qudaTargets.cmake"
-  NAMESPACE quda::)
-
-set(ConfigPackageLocation lib/cmake/quda/)
 
-install(
-  EXPORT qudaTargets
-  NAMESPACE quda::
-  DESTINATION ${ConfigPackageLocation})
 
 add_custom_target(
   mpi_nvtx
```

Comment:
Good practice to:

add_library( QUDA::quda ALIAS quda )

and have the tests link to QUDA::quda ?